docker/Dockerfile

```diff
@@ -35,11 +35,6 @@ RUN ./mvnw -q --batch-mode -DskipTests --also-make -pl zipkin-server install
 RUN mkdir -p /zipkin && cp zipkin-server/target/zipkin-server-*-exec.jar /zipkin && cd /zipkin && jar xf *.jar && rm *.jar
 RUN mkdir -p /zipkin-slim && cp zipkin-server/target/zipkin-server-*-slim.jar /zipkin-slim && cd /zipkin-slim && jar xf *.jar && rm *.jar
 
-FROM maven:3-openjdk-14-slim as zipkin-builder
```

Comment:
I couldn't see where this layer was used..

Reply:
It's used here in a hook

https://github.com/openzipkin/zipkin/blob/206abbc26c333f974e09a08063995f4a23af0a39/docker/hooks/post_push#L38

It's the only way to have these cache across builds with docker hub that I could find.

Reply:
ic, I was actually wondering why we are explicitly caching in the image when we can do stuff like that in travis and github actions..

something to think about, and don't have a better plan at the moment. it is a pity to have copy/pasta..